nss-cohort-36/bangazon-client-virtual-magicians-1
# Sp user profile

# Description
`User Profile` component is built. When you click **My Profile** in the NavBar, you should be able to view your profile with your name, username, email, and two buttons on it. The buttons don't do anything yet.

Closes https://github.com/nss-cohort-36/bangazon-api-virtual_magicians/issues/54
## Type of change

Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout sp-user-profile`
4. Make sure you are registered as a user and try to view My Profile.
5. Also check to make sure the fetch calls in `APIManager` now include auth tokens

# Checklist:
